internal/kafka/internal/services/util.go

```diff
@@ -72,3 +73,11 @@ func BuildKeycloakClientNameIdentifier(kafkaRequestID string) string {
 func BuildCustomClaimCheck(kafkaRequest *dbapi.KafkaRequest) string {
 	return fmt.Sprintf("@.rh-org-id == '%s'|| @.org_id == '%s'", kafkaRequest.OrganisationId, kafkaRequest.OrganisationId)
 }
+
+func ParseSize(sizeId string) (int64, error) {
+	size, err := strconv.ParseInt(strings.TrimPrefix(sizeId, "x"), 10, 64)
+	if err != nil {
+		return 0, err
+	}
+	return size, nil
```

Comment:
Same as comment above, instead of parsing the size from the id let's use the `CapacityConsumed` property in the config.